src/e2e-browser/.env.defaults

```diff
@@ -1,10 +1,14 @@
-# Save a copy of this file as .env.local to prevent your credentials from being committed
+# Save a copy of this file as .env.local to prevent your credentials from being
```

Comment:
My PR also added explanatory text around all this - should probably just merge the text of both I think, but this reference to `.env.local` is still wrong though:
```suggestion
# Save a copy of this file as .env.test.local to prevent your credentials from being
```

Reply:
Ah found them. I merged them into here, although I removed the remark about the resource - the tests do assume them to be the Pod root. 7d5bcfd6